src/main/java/Ui.java

```diff
@@ -0,0 +1,65 @@
+import java.util.Scanner;
+
+public class Ui {
+
+    private static final String LOGO =
+            " \n____        _        \n"
+                    + "|  _ \\ _   _| | _____ \n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
+
+    private static final String DIVIDER =
+            "    ____________________________________________________________\n";
+
+//    private final Scanner sc;
+//
+//    Ui() {
+//        sc = new Scanner(System.in);
+//    }
+//
+//    String readCommand() {
+//        return sc.nextLine();
+//    }
+
+    String showDivider() {
+        return DIVIDER;
+    }
+
+    String showError(DukeException e) {
+        return "     " + e.getMessage();
+    }
+
+    String showWelcome() {
+        return "Hello from\n" + LOGO
+                + showDivider()
+                + "     Hello! I'm Duke!\n" + "     What can I do for you?\n"
+                + showDivider();
+    }
+
+    String showGoodbye() {
+        return "     Bye. Hope to see you again soon!\n";
+    }
+
+    String showTaskList(String taskListString) {
+        return "     Here are the tasks in your list:\n" +  taskListString;
+    }
+
+    String showDoneTask(Task task) {
+        return "     Nice! I've marked this task as done:\n       " + task;
+    }
+
+    String showDeleteTask(Task task, int listLength) {
+        return "     Noted. I've removed this task:\n       " + task 
+                + "\n     Now you have " + listLength + " tasks in the list.";
+    }
+
+    String showAddedTask(Task task, int listLength) {
+        return "     Got it. I've added this task:\n       " + task +
+                "\n     Now you have " + listLength + " tasks in the list.";
```

Comment:
Here is a small personal preference. I would suggest using a ternary operator so that "task" would be used instead of "tasks" if listLength is 0 or 1.

In addition, may I point out that the plus operator is not in front of the next line? I noticed an inconsistency within this method in comparison to showDeleteTask and showWelcome.